proptest-state-machine/src/strategy.rs

```diff
@@ -567,7 +567,7 @@ mod test {
     ///
     /// This constant can be determined from the test
     /// `number_of_sequential_value_tree_simplifications`.
-    const SIMPLIFICATIONS: usize = 699;
+    const SIMPLIFICATIONS: usize = 32;
```

Comment:
for clarity why this change?

Reply:
I missed this change in #392 that fixed a bug in shrinking, it was actually failing on that branch